skrub/_dataframe/tests/test_common.py

```diff
@@ -122,6 +125,16 @@ def test_make_dataframe_like(df_module, example_data_dict):
     assert ns.dataframe_module_name(df) == df_module.name
 
 
+def test_make_dataframe_like_pandas_index():
```

Comment:
It could be worth to add a little test for the reset function for completeness and direct testing.